inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -252,7 +253,7 @@ unique(study_data_raw$q4_q6_sex)
 #  function)
 study_data_raw %>% 
   filter(q137_q35_died_date < q88_q33_born_date) %>% 
-  select(matches("fact_0_id"))
+  select("fact_0_id")
```

Comment:
Why did you remove the matches here?

Reply:
just not sure it adds much, and extra layer which might confuse people... ? 
(Really the names should just be fixed..... but thats a different story)

Reply:
If the "matches" isn't there, then the script throws an error unless the column exists. That's why I put it there.